src/skate.js

```diff
@@ -84,7 +84,7 @@ function skate (id, definition) {
   definition = inherit(definition || {}, skate.defaults);
   definition.id = id;
   if (definition.prototype === skate.defaults.prototype) {
-    definition.prototype = {};
+    definition.prototype = Object.create(HTMLElement.prototype);
```

Comment:
I like the latter solution. Simple and provides a lot of convenience and prevents potential gotchas with how the spec and native implementations handle things. It's unexpected that a custom element wouldn't implicitly inherit from HTMLElement. It is, after all, an element. Even `HTMLUnknownElement` derives from `HTMLElement`.